sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosAsyncContainer.java

```diff
@@ -1473,6 +1475,15 @@ public String getId() {
         return withContext(context -> deleteItemInternal(itemId, null, requestOptions, context));
     }
 
+    public Mono<CosmosItemResponse<Object>> deleteAllItemsByPartitionKeyAsync(PartitionKey partitionKey, CosmosItemRequestOptions options) {
```

Comment:
`deleteAllItemsByPartitionKeyAsync` or `deleteAllItemsByPartitionKey`?

please check other APIs in this class, follow the same naming pattern

Reply:
Replaced with deleteAllItemsByPartitionKey